src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return 40 * days - 50;
```

Comment:
```suggestion
    return (40 * days) - 50;
```
use parentheses or move this equation to the variable